stdlib/3/smtplib.pyi

```diff
@@ -1,25 +1,29 @@
-from typing import Any
+from email.message import Message as _Message
+from typing import Any, AnyStr, Dict, Generic, List, Optional, Tuple, Union
+
+_Reply = Tuple[int, bytes]
+_SendErrs = Dict[str, _Reply]
 
 class SMTPException(OSError): ...
 class SMTPServerDisconnected(SMTPException): ...
 
 class SMTPResponseException(SMTPException):
-    smtp_code = ...  # type: Any
-    smtp_error = ...  # type: Any
-    args = ...  # type: Any
-    def __init__(self, code, msg) -> None: ...
+    smtp_code = ...  # type: int
+    smtp_error = ...  # type: Union[bytes, str]
+    args = ...  # type: Tuple[int, Union[bytes, str]]
+    def __init__(self, code: int, msg: Union[bytes, str]) -> None: ...
 
 class SMTPSenderRefused(SMTPResponseException):
-    smtp_code = ...  # type: Any
+    smtp_code = ...  # type: int
     smtp_error = ...  # type: Any
```

Comment:
These don't need to be Any, do they?

Reply:
Nope, they don't. However, now args is typed correctly, I'm getting:

```
stdlib/3/smtplib.pyi:21: error: Incompatible types in assignment (expression has type "Tuple[int, bytes, str]", base class "SMTPResponseException" defined the type as "Tuple[int, Union[bytes, str]]")
```

How should I fix this? Make the superclass type a Union of the two possible types?

Reply:
That's one option; I'd also be OK with making them `Tuple[Any, ...]` since the precise types for `args` don't seem that important.

Reply:
As there are only a couple of possible options (and it's fairly unlikely that new subclasses will appear), I went with the union option.